packages/zowe-explorer/src/extension.ts

```diff
@@ -122,6 +122,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
                 JSON.stringify(err)
         );
     }
+    // set a command to silently reload extension
+    context.subscriptions.push(
+        vscode.commands.registerCommand("zowe.extRefresh", async () => {
+            await deactivate();
+            for (const sub of context.subscriptions) {
+                try {
+                    await sub.dispose();
+                } catch (e) {
+                    globals.LOG.error(e);
+                }
+            }
+            await activate(context);
+        })
+    );
```

Comment:
@zFernand0 the subscriptions to all the other commands allow the loop here to dispose of them before reactivating.  If they aren't disposed errors will pop up stating that the commands are already registered and in our case they are registered with the previous treeProvider.

Reply:
thanks! 😋 
that helps! : )